spacy/errors.py

```diff
@@ -253,6 +253,9 @@ class Errors(object):
     E098 = ("Trying to set conflicting doc.ents: '{span1}' and '{span2}'. A token"
             " can only be part of one entity, so make sure the entities you're "
             "setting don't overlap.")
+    E099 = ("The splitted token can only have one root (head = 0).")
+    E100 = ("The splitted token needs to have a root (head = 0)")
```

Comment:
Hmm. `splitted` isn't ideal English, but it's less ambiguous than `The split token...`, so I think I prefer it.

Reply:
This is referring to the resulting split tokens, right? So how about something like "The new tokens created by splitting a token..."?

Reply:
Maybe "the newly split tokens"?